include/GafferScene/ScenePlug.h

```diff
@@ -137,6 +137,9 @@ class GAFFERSCENE_API ScenePlug : public Gaffer::ValuePlug
 			PathScope( const Gaffer::Context *context );
 			PathScope( const Gaffer::Context *context, const ScenePath &scenePath );
 
+			PathScope( const Gaffer::ThreadState &threadState );
```

Comment:
Are these always to be used in preference to the context equivalents? If so, should we remove (or deprecate) the old ones? Or if not, can we have a doc comment explaining when you'd use one over the other?

Reply:
No, as mentioned in the ThreadState docs, you only need to worry about ThreadState when you're spawning TBB tasks. I've added some additional comments to this effect in https://github.com/GafferHQ/gaffer/pull/3196/commits/d530a2f37f4a1ac6b54d50afa30c3b67f26b0ed0.